packages/test-base/src/androidTest/kotlin/io/realm/kotlin/test/shared/QueryTests.kt

```diff
@@ -141,6 +142,10 @@ class QueryTests {
                 }
                 RealmStorageType.OBJECT_ID -> {
                     realm.query<QuerySample>("objectIdField = $0", ObjectId.from("507f191e810c19729de860ea"))
+                    realm.query<QuerySample>("bsonObjectIdField = $0", ObjectId.from("507f191e810c19729de860ea"))
+
+                    realm.query<QuerySample>("objectIdField = $0", BsonObjectId("507f191e810c19729de860ea"))
```

Comment:
Copy/paste mistake?

Reply:
We are testing the combination of the two `ObjectId` query arguments on the two fields defined by the different `ObjectId` implementations. They should be interchangeable.